scd30/scd30.h

```diff
@@ -75,15 +77,16 @@ u8 scd_get_configured_address(void);
  *                              ambient pressure compensation (reverts to
  *                              altitude compensation, if set), 700-1200mBar
  *                              allowable range otherwise
- * @return                      0 if the command was successful, an error code
+ *
+ * Return:                      0 if the command was successful, an error code
  *                              otherwise
  */
 s16 scd_start_periodic_measurement(u16 ambient_pressure_mbar);
 
 /**
  * scd_stop_periodic_measurement() - Stop the continuous measurement
  *
- * @return  0 if the command was successful, else an error code
+ * Return:  0 if the command was successful, else an error code
```

Comment:
Why switch from the `@return` doxygen keyword to `Return:`?

Reply:
Very good point. I always assumed that 'Return' is the doxygen standard because it's used in the Linux kernel but https://www.kernel.org/doc/html/latest/doc-guide/kernel-doc.html?highlight=doxygen has a big fat note stating that

> The kernel-doc format is deceptively similar to javadoc, gtk-doc or Doxygen, yet distinctively different, for historical reasons

At least it's now consistent in embedded-scp. embedded-sgp also uses Kernel style `Return:` but embedded-sht uses doxygen's `@return`.

I'll change it back to doxygen's `@return`.